src/components/MAPI/apps/ClusterDetailAppListItem.tsx

```diff
@@ -24,7 +24,7 @@ const StyledBox = styled(Box)`
 `;
 
 const Header = styled(Box)`
-  &[aria-disabled] {
+  &[aria-disabled='true'] {
```

Comment:
I missed this in a previous PR. It makes the app always be displayed as disabled